src/components/timeline/Markers.js

```diff
@@ -410,6 +411,30 @@ class TimelineMarkersImplementation extends React.PureComponent<Props, State> {
     });
   };
 
+  _stopPropagation(e: TransitionEvent) {
```

Comment:
Optional: This could be a separate function instead of a method since it doesnt need anything from the component. Normally methods passed to an event handler need their `this` bound, so I think this could be confusing.